Packs/SophosCentral/Integrations/SophosCentral/SophosCentral.py

```diff
@@ -637,17 +679,27 @@ def create_alert_output(item: Dict, table_headers: List[str]) -> Dict[str, Optio
         object_data (dict(str)): The output dictionary.
     """
     alert_data = {field: item.get(field) for field in table_headers + ['groupKey', 'product']}
+
     managed_agent = item.get('managedAgent')
     if managed_agent:
+        managed_agent_morphed_id = flip_chars(managed_agent.get('id', ''))
         alert_data['managedAgentId'] = managed_agent.get('id')
+        alert_data['managedAgentIdMorphed'] = managed_agent_morphed_id

```

Comment:
Do you want it to be an empty string while the regular ID can be `None`?

Reply:
I think this is fine, as the id will always exist as long as the object is returned from the API.